src/common/snippets/docs/snippets_design_guide.md

```diff
@@ -647,14 +647,14 @@ So if a loop's `work_amount` is not evenly divisible by its `increment`, it mean
 4. `OptimizeLoopSingleEvaluation` moves all pointer arithmetic to finalization offsets in `LoopEnd`, and marks the loops that will be executed only once.
 This information will be used during code emission to eliminate redundant instructions.
 
-Please see [assign_registers.cpp](../src/lowered/pass/assign_registers.cpp) and [insert_tail_loop.cpp](../src/lowered/pass/insert_tail_loop.cpp) for more info regarding the main passes in the `Preparation` stage. 
+Please see [assign_registers.cpp](../src/lowered/pass/assign_registers.cpp) for more info regarding the main passes in the `Preparation` stage. 
```

Comment:
insert_tail_loop is now called InsertSpecificIterations because it can insert not only the last iteration, but also the first one (i.e. to initialize accumulators). Please correct the bullet name on L641 as well.
```suggestion
Please see [assign_registers.cpp](../src/lowered/pass/assign_registers.cpp) and and [insert_specific_iterations.cpp](../src/lowered/pass/insert_specific_iterations.cpp) for more info regarding the main passes in the `Preparation` stage. 
```

Reply:
Done